sdks/go/alpha.go

```diff
@@ -96,7 +96,9 @@ func (a *Alpha) GetCounterCount(key string) (int64, error) {
 }
 
 // IncrementCounter increases a counter by the given nonnegative integer amount.
-// Will execute the increment operation against the current CRD value. Will max at max(int64).
+// Will execute the increment operation against the current CRD value.
+// By default, the counter's capacity is set to 1000. Attempting to set the capacity
```

Comment:
We probably don't need to reference this here since this is for incrementing the count of the Counter.